src/Microsoft.SqlTools.ServiceLayer/Scripting/ScripterCore.cs

```diff
@@ -273,24 +273,28 @@ internal DefinitionResult GetDefinitionUsingDeclarationType(DeclarationType type
                 string schemaName,
                 string objectType)
         {
-            StringCollection scripts = sqlScriptGetter(objectName, schemaName, null);
+            bool scriptSuccess = false;
```

Comment:
>bool scriptSuccess = false; [](start = 12, length = 27)

I would just declare this variable where it's first used on line #281..there's no need for it to be in scope as false at this point that I can see.